docs/website/versioned_docs/version-3.0.0/user-guides/quickstart.md

```diff
@@ -63,11 +63,46 @@ $ npx express-generator
      $ DEBUG=express:* npm start
 ```
 
-Your source code has now been generated and created in the directory.
 
 </TabItem>
+  <TabItem value="dotnet" label=".NET">
+
+  For .NET we will use the [ASP.NET Core MVC](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/start-mvc?view=aspnetcore-6.0&tabs=visual-studio-code) example. 
+
+  ASP.NET MVC is a web application framework that implements the model-view-controller (MVC) pattern.
+
+  1. Generate an example project:
+
+```console
+$ dotnet new mvc
+
+Welcome to .NET 6.0!
+---------------------
+SDK Version: 6.0.104
+
+...
+
+The template "ASP.NET Core Web App (Model-View-Controller)" was created successfully.
+This template contains technologies from parties other than Microsoft, see https://aka.ms/aspnetcore/6.0-third-party-notices for details.
+
+Processing post-creation actions...
+Running 'dotnet restore' on /Users/user/dotnet/dotnet.csproj...
+  Determining projects to restore...
+  Restored /Users/user/dotnet/dotnet.csproj (in 84 ms).
+Restore succeeded.
+```
+
+2. Rename the `dotnet.csproj` file to `app.csproj`:
```

Comment:
Just curious, why is this needed? Does this mean that even existing projects need to have their `.csproj` file renamed as well?

If this is really needed, maybe we can skip this operation here by generating the project with the right `app.csproj` file? For example with this command:

```
dotnet new mvc --name app
```

Reply:
Whoops! You are right.